src/configuration.ts

```diff
@@ -120,10 +120,9 @@ function findNextVersion(rootPath: string): string | undefined {
 export function findRepoFromPkg(pkg: any): string | undefined {
   const url = pkg.repository.url || pkg.repository;
   const normalized = normalize(url).url;
-  const match = normalized.match(/github\.com[:/]([^./]+\/[^./]+)(?:\.git)?/);
+  const match = normalized.match(/github\.com[:\/]([^\/]+\/[^\/]+?)(?:\.git)?$/);
```

Comment:
why the extra escaping?

Reply:
uh! My bad was testing it online and pushed it.